src/Bookings.js

```diff
@@ -1,18 +1,45 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    let filteredBookings = [];
+    bookings.forEach(booking => {
+      if (
+        booking.firstName.toLowerCase().includes(searchVal.toLowerCase()) ||
+        booking.surname.toLowerCase().includes(searchVal.toLowerCase())
+      ) {
+        filteredBookings.push(booking);
+      }
+    });
+    setBookings(filteredBookings);
   };
 
+  const [bookings, setBookings] = useState(0);
+  const [profile, setProfile] = useState(0);
```

Comment:
rather than setting your state as 0 for no data, try and use `null` or `[]` instead, I noticed you're using `profile` for the customerId, but if a customer had an id of 0 this could cause problems down the line